contracts/router/modules/pool/gmx/GMXV1StableArbitrumModule.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import {GMXV1Module} from "./GMXV1Module.sol";
+
+contract GMXV1StableArbitrumModule is GMXV1Module {
+    address public constant USDC_E = 0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8;
```

Comment:
Add a comment that the order of tokens is the same as within GMX Vault's `allWhitelistedTokens`?